cms/utils/moderator.py

```diff
@@ -1,26 +0,0 @@
-
```

Comment:
May need to document / deprecate the removal of these methods. One to document somewhere, potentially the Release document.

Reply:
I have started a Deprecation PR for v3. Will add those into it.